### PR TITLE
Added redis config for port and cache prefix.

### PR DESCRIPTION
**Pull request by thom8**
Allow overriding of the redis port and cache prefix without affecting current behaviour.

Projects will be able to customise the redis HOST, PORT and CACHE_PREFIX if required.

### CODE REVIEW

File: .docker/images/govcms8/settings/settings.php [hunk 1]
```diff
@@ -145,9 +145,9 @@ public function __toString() {
 if (getenv('LAGOON') && (getenv('ENABLE_REDIS'))) {
   $settings['redis.connection']['interface'] = 'PhpRedis';
   $settings['redis.connection']['host'] = getenv('REDIS_HOST') ?: 'redis';
-  $settings['redis.connection']['port'] = 6379;
+  $settings['redis.connection']['port'] = getenv('REDIS_PORT') ?: 6379;
```
Comment (srowlands) [inline .docker/images/govcms8/settings/settings.php]: Looks good, but Lagoon env var for `REDIS_PORT` includes more than it should (e.g looks to be `REDIS_PORT=tcp://172.30.42.136:6379`)

Comment (thom8) [reply .docker/images/govcms8/settings/settings.php]: Good catch, thanks! will update to `REDIS_SERVICE_PORT`

**Comment by simesy** [comment]
Can such changes please start in https://github.com/govCMS/scaffold-tooling which is canonical. Also https://github.com/govCMS/govcms8lagoon/issues/97

**Comment by srowlands** [comment]
Merged in #100

**Pull request closed without merging**